assets/styling.css

```diff
@@ -1,228 +1,228 @@
 /* --- FONTS START ---- */
 @font-face {
     font-family: 'Brown';
-    src: url('/font/Brown-RegularItalicAlt.eot');
-    src: local('/font/Brown-RegularItalicAlt'),
-        url('/font/Brown-RegularItalicAlt.eot?#iefix') format('embedded-opentype'),
-        url('/font/Brown-RegularItalicAlt.woff') format('woff'),
-        url('/font/Brown-RegularItalicAlt.ttf') format('truetype');
+    src: url('https://www.katacoda.com/daml/assets/Brown-RegularItalicAlt.eot');
```

Comment:
Why do you use absolute paths? I think just `Brown-RegularItalicAlt.eot` _should_ do, but let's try like this.
The problem with your original version was the leading `/`, which makes the path relative to root.